packages/stargate/src/modules/staking/aminomessages.ts

```diff
@@ -143,6 +144,20 @@ export function isAminoMsgUndelegate(msg: AminoMsg): msg is AminoMsgUndelegate {
   return msg.type === "cosmos-sdk/MsgUndelegate";
 }
 
+export interface AminoMsgCancelUnbondingDelegation extends AminoMsg {
+  readonly type: "cosmos-sdk/MsgCancelUnbondingDelegation";
+  readonly value: {
+    readonly delegator_address: string;
+    readonly validator_address: string;
+    readonly amount: Coin;
+    readonly creation_height: bigint;
```

Comment:
In Amino JSON we cannot use Bigints. Do you know how this is actually implemented in Cosmos SDK? Strings?

Reply:
Cosmos SDK implemented creation_height is int64, so string can be used here, right? 
https://github.com/cosmos/cosmos-sdk/blob/9fd3d13e2c32b64af4abd0672d1f25c2477c6e22/x/staking/proto/cosmos/staking/v1beta1/tx.proto#L184

Reply:
changed to string